snmp/datadog_checks/snmp/data/profiles/apc_ups.yaml

```diff
@@ -125,17 +176,21 @@ metrics:
         column:
           OID: 1.3.6.1.4.1.318.1.1.1.12.1.2.1.2
           name: upsOutletGroupStatusName
-
 metric_tags:
   - tag: model
-    OID: 1.3.6.1.4.1.318.1.1.1.1.1.1.0
+    OID: 1.3.6.1.4.1.318.1.1.1.1.1.1.0  # The UPS model name (e.g. 'APC Smart-UPS 600', 'Smart-UPS RT 6000 RM XL').
     symbol: upsBasicIdentModel
   - tag: serial_num
     OID: 1.3.6.1.4.1.318.1.1.1.1.2.3.0
-    symbol: upsAdvIdentSerialNumber
+    symbol: upsAdvIdentSerialNumber     # An 8-character string identifying the serial number of
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
I think there is a `description` field for OID symbols, that could be used instead of comments to add context to an instance of metric tags

Reply:
I'm not completely sure if there are advantages to use `description` field instead of comments since the content is not use.

I think we can discus about it (not part of this PR) since that apply to all profiles and might need a refactoring of all profiles.

Reply:
Re-phrasing my comment: I don't think we have a standard, our profiles do not have `description` fields, but the generator is creating profiles with `description`. Not sure this deserves an RFC, maybe yes. https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/generate_profile.py#L239-L241

Reply:
If de-facto standard is using comments, ok for treating it out of this PR

Reply:
Not sure we need to migrate all profiles to an eventual new standard.

Reply:
It's probably not high prio, but I think it can worth discussing about it. Beside description, there is also a `MIB` that is used as documentation for corecheck (it was useful for python check that did MIB parsing).